src/Chapter13/Listing13.25.ExamingingAnExpressionTree.cs

```diff
@@ -56,5 +57,5 @@ expression.NodeType switch
                 _ => expression.ToString() +
                     " (" + expression.NodeType.ToString() + ")",
             };
-    }
-}
\ No newline at end of file
+        #endregion INCLUDE
+    }
```

Comment:
It looks like a `}` was dropped here, causing the failures.

Reply:
Fixed this by https://github.com/IntelliTect/EssentialCSharp/pull/280/commits/66456695704d90fe6abd93f77353b52db85534af

Just adding in a curly brace } to the end

Reply:
@HannaMichaelis